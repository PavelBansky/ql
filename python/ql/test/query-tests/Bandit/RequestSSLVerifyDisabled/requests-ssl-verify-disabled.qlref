Bandit/requests-ssl-verify-disabled.ql