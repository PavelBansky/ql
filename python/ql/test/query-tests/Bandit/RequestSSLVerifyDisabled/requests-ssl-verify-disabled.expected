| Test.py:4:1:4:47 | Attribute() | Requests call with verify=False disabling SSL certificate checks, security issue. |
| Test.py:6:1:6:48 | Attribute() | Requests call with verify=False disabling SSL certificate checks, security issue. |
| Test.py:8:1:8:47 | Attribute() | Requests call with verify=False disabling SSL certificate checks, security issue. |
| Test.py:10:1:10:50 | Attribute() | Requests call with verify=False disabling SSL certificate checks, security issue. |
| Test.py:12:1:12:49 | Attribute() | Requests call with verify=False disabling SSL certificate checks, security issue. |
| Test.py:14:1:14:51 | Attribute() | Requests call with verify=False disabling SSL certificate checks, security issue. |
| Test.py:16:1:16:48 | Attribute() | Requests call with verify=False disabling SSL certificate checks, security issue. |
